.circleci/integration-tests/master_dag.py

```diff
@@ -133,6 +133,7 @@ def prepare_dag_dependency(task_info, execution_time):
         {"big_query_sensor_dag": "example_bigquery_sensors"},
         {"dataproc_dag": "example_gcp_dataproc"},
         {"kubernetes_engine_dag": "example_google_kubernetes_engine"},
+        {"bigquery_dag_impersonation": "example_async_bigquery_impersonation"},
```

Comment:
```suggestion
        {"bigquery_impersonation_dag": "example_bigquery_impersonation"},
```

to be inline with other examples which end with _dag

Reply:
Fixed